### PR TITLE
Team views

### PR DESCRIPTION
**Pull request by florisdobber**
Currently uses SVG for icons, can be converted later to bootstrap icons. Not all icons are implemented yet, but the general look is finished. See below for different views.
Owner:
owner
Member:
member
Prospect:
prospect

### CODE REVIEW

File: src/views/TeamView.vue [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<template>
+  <div>
+      <h1>{{ name }}  <img v-if="permission >= 2" src="../../assets/edit-icon.svg" alt="edit"></h1>
```
Comment (jackblanc) [inline src/views/TeamView.vue:3]: Can we use constants for permissions? We probably should have started that earlier on...

File: src/views/TeamView.vue [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      members: ['MEMBER 1', 'MEMBER 2', 'MEMBER 3', 'MEMBER 4'],
+      progress: 12,
+      goal: 20,
+      target: new Date('2020-03-25'),
+      permission: 1, // 0 prospect, 1 member, 2 owner
+    };
+  },
+  computed: {
+    // format the target date into the appropriate format
+    targetDate() {
```
Comment (jackblanc) [inline src/views/TeamView.vue:51]: what does "targetDate" mean?

File: src/views/TeamView.vue [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        class="member">{{ member }}</p>
+      </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TeamView',
+  data() {
+    return {
```
Comment (jackblanc) [inline src/views/TeamView.vue:39]: Please make sure this matches the API spec to avoid future complications

File: src/views/TeamView.vue (hunk 1 above)
Comment (gabriel-peter) [reply src/views/TeamView.vue:3]: Agreed!

**Pull request merged**